docs/_Docs_GetStarted.md

```diff
@@ -68,78 +73,79 @@ The final step for installation is copying over some of the files from our submo
 yarn run provision:token:contracts
 ```
 
-### Testing
+## `glider-rc.1` on the Görli testnet
 
-To deploy all contracts and run all tests, run the following command.
+The [Glider release candidate](/colonynetwork/docs-releases/) is in many ways a simpler and easier way to experiment than setting up a local development environment, and can be very useful if you're looking to just get a sense of how the colonyNetwork contracts work, or want to build extensions/integrations that remain inside the EVM.
 
-```
-yarn run test:contracts
-```
+To connect, you'll need to know the address of the colonyNetwork (which is, in reality, the address of the `etherRouter` contract; see [The Delegate Proxy Pattern](/colonynetwork/docs-the-delegate-proxy-pattern/) for more info).
 
-### Development
+`ColonyNetwork`: `0x79073fc2117dD054FCEdaCad1E7018C9CbE3ec0B`
 
-Alternatively, you can start a local test node and deploy the contracts yourself using the locally installed `truffle` package.
+You will also require Görli test ETH, and a deployed ERC20 token to import.
```

Comment:
If they plan on creating a Colony and interacting with it they will need a lot more than that 👆 . The following set of transactions are a useful reference as to how to setup and wire a new Colony. 
https://github.com/JoinColony/colonyNetwork/issues/607

The ENS registration steps are optional (as they are a user friendly UI used by the dApp) and the Extension Contracts steps might also be skipped if Payments aren't a required functionality or Old style permissions aren't needed.